### PR TITLE
Add Login Page

### PR DESCRIPTION
**Pull request by jaminko**

### CODE REVIEW

File: DepositeCalcTests/Tests/LoginPageTests.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            String currentURL = driver.Url;
+
+            // Act
+            loginFld.SendKeys("test");
+            passworldFld.SendKeys("newyork1");
+            loginBtn.Click();
+
+            // Assert
+            Assert.IsTrue(driver.FindElement(By.Id("amount")).Displayed);
+            Assert.AreNotEqual(currentURL, driver.Url);
```
Comment (varajan) [inline DepositeCalcTests/Tests/LoginPageTests.cs]: I think it is better to verify the current URL is equal to '.../Calculator'

Comment (jaminko) [reply DepositeCalcTests/Tests/LoginPageTests.cs]: Got it!
In my opinion, we should also update the README.MD file:
row 10 -  change * Login filed, labeled 'User' to  * Login filed, labeled 'User:'
row 11 -  change * Password filed, labeled 'Password'change to  * Password filed, labeled 'Password:'

File: DepositeCalcTests/Tests/LoginPageTests.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/";
```
Comment (varajan) [inline DepositeCalcTests/Tests/LoginPageTests.cs:21]: фівадфіва жо жаофівжао 
```suggestion
            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(40);
            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(65);
            driver.Url = "https://localhost:5001/";
```

File: DepositeCalcTests/Tests/LoginPageTests.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        [TestCase("test", "", "Incorrect password!")]
+        [TestCase("", "newyork1", "Incorrect user name!")]
+        [TestCase("", "", "User not found!")]
+        public void NegativeTest(string login, string password, string expectedErrMsg)
+        {
+            // Arrange
+            var loginPage = new LoginPage(driver);
+
+            // Act
+            loginPage.Login(login, password);
```
Comment (varajan) [inline DepositeCalcTests/Tests/LoginPageTests.cs:39]: укепвфуі емуцк ек

**Review by varajan — DISMISSED** [review]

**Review by varajan — APPROVED** [review]

**Pull request merged**